### PR TITLE
Account settings page

### PR DESCRIPTION
**Pull request by almathew**
all tests pass.

### CODE REVIEW

**Comment by almathew** [comment]
Works of art:

microsites_account_settings

microsite_account_delete

**Comment by aspidites** [comment]
But what about modals? Do they work now?

**Comment by aspidites** [comment]
2015-01-02-160236_1916x1056_scrot
Answered my own question.

**Comment by aspidites** [comment]
Yay. Modals look normal now:
2015-01-02-162650_1916x1056_scrot

**Comment by aspidites** [comment]
@almathew are accordions broken, or did you decide to remove them after all?

**Comment by almathew** [comment]
Decided to remove them for now.

File: static/dseo_account_settings.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    font-size: 30px;
+    line-height: 36px;
+}
+
+#direct_container .account-settings-list ul li {
+    list-style: disc;
+}
+
+#direct_container .account-settings-list ul {
+    margin-left: 4%;
```
Comment (aspidites) [inline static/dseo_account_settings.css]: I know bootstrap uses percentages, but do we? I thought we used px for everything.

Comment (almathew) [inline static/dseo_account_settings.css]: Technically, no? I've been switching for microsites with the rollout of bootstrap 3. Also, it looks better in mobile.

**Pull request merged**